<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="C365L0041447901" name="com.ortusolis.registration.LMrqstAprvByAd" tests="3" failures="0" timestamp="18 Jun 2018 06:11:28 GMT" time="5.811" errors="2">
  <testcase name="LM_UpdateValues" time="0.144" classname="com.ortusolis.registration.LMrqstAprvByAd">
    <error type="java.lang.ArrayIndexOutOfBoundsException" message="0">
      <![CDATA[java.lang.ArrayIndexOutOfBoundsException: 0
	at com.ortusolis.utilities.CommonUtilities.ufSplitMe(CommonUtilities.java:35)
	at com.ortusolis.utilities.SeleniumUtilities.loadWithBy(SeleniumUtilities.java:18)
	at com.ortusolis.pageobjectsPO.LMrqstAprvByAdmnPO.RqstAprvPage(LMrqstAprvByAdmnPO.java:71)
	at com.ortusolis.pageobjectsPO.LMrqstAprvByAdmnPO.RqstAprvLocators(LMrqstAprvByAdmnPO.java:37)
	at com.ortusolis.registration.LMrqstAprvByAd.LM_UpdateValues(LMrqstAprvByAd.java:46)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- LM_UpdateValues -->
  <testcase name="LM_AprvShikesho" time="0.471" classname="com.ortusolis.registration.LMrqstAprvByAd"/>
  <testcase name="LM_AdmnLog" time="5.196" classname="com.ortusolis.registration.LMrqstAprvByAd">
    <error type="java.lang.ArrayIndexOutOfBoundsException" message="0">
      <![CDATA[java.lang.ArrayIndexOutOfBoundsException: 0
	at com.ortusolis.utilities.CommonUtilities.ufSplitMe(CommonUtilities.java:35)
	at com.ortusolis.utilities.SeleniumUtilities.loadWithBy(SeleniumUtilities.java:18)
	at com.ortusolis.pageobjectsPO.LMrqstAprvByAdmnPO.RqstAprvPage(LMrqstAprvByAdmnPO.java:71)
	at com.ortusolis.pageobjectsPO.LMrqstAprvByAdmnPO.RqstAprvLocators(LMrqstAprvByAdmnPO.java:37)
	at com.ortusolis.registration.LMrqstAprvByAd.LM_AdmnLog(LMrqstAprvByAd.java:21)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- LM_AdmnLog -->
</testsuite> <!-- com.ortusolis.registration.LMrqstAprvByAd -->
